tada_a_bigoh.py

```diff
@@ -37,7 +41,7 @@
         # save the directory containing functions to be analyzed
         save.save_directory(constants.DIRECTORY, tada_arguments.directory)
         # perform the small doubling experiment
-        while current_size <= constants.SIZE_STOP:
+        while indicator >= 0.1 and last_last_size != current_size:
```

Comment:
I like the use of the indicator feature!

Reply:
Me too! It looks great!